locales/nl_NL.json

```diff
@@ -1,23 +1,23 @@
 {
-  "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
-  "buy": "buy",
-  "sell": "sell",
-  "continue": "Continue",
+  "sandbox-warning": "Dit is een sandbox-omgeving voor het testen van de API-integratie. Stuur hier nooit echt geld naar toe!",
+  "buy": "kopen",
+  "sell": "verkopen",
+  "continue": "Doorgaan",
   "ok": "OK",
-  "back": "Back",
+  "back": "Terug",
   "help": "Help",
-  "and": "and",
-  "from": "From",
-  "to": "To",
-  "or": "or",
-  "price": "Price",
+  "and": "en",
+  "from": "Van",
+  "to": "Naar",
+  "or": "of",
+  "price": "Prijs",
   "fee": "Fee",
-  "try-again": "Try again",
-  "refresh": "Refresh",
-  "finish": "Finish",
-  "price-each": "{{price}} each",
+  "try-again": "Probeer opnieuw",
+  "refresh": "Vernieuwen",
+  "finish": "Beëindigen",
```

Comment:
Context ontbreekt hier.

Reply:
You should contact me on Telegram if you have questions: @cstfenwick

These are generic buttons that appear in numerous places.